curriculum/challenges/english/07-scientific-computing-with-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/656877f71bba2b97acedf9af.md

```diff
@@ -15,7 +15,7 @@ str.maketrans({'t': 'c', 'l': 'b'})
 
 The above, when called on a string, will replace all `t` characters with `c` and all `l` characters with `b`.
```

Comment:
I'm not sure if we should use quotes around single characters. Also I don't think we should treat the characters above differently from `-` and ` `.

Reply:
In my opinion, it would be better to remove quotes in this case.